src/util/rocksdb.rs

```diff
@@ -84,16 +95,16 @@ fn check_and_open(path: &str,
 
     // List all column families in current db.
     let cfs_list = try!(DB::list_column_families(&db_opt, path));
-    let existed: HashSet<&str> = cfs_list.iter().map(|v| v.as_str()).collect();
-    let needed: HashSet<&str> = cfs_opts.keys().map(|v| *v).collect();
+    let existed: Vec<&str> = cfs_list.iter().map(|v| v.as_str()).collect();
+    let needed: Vec<&str> = cfs_opts.iter().map(|cf_options| cf_options.cf).collect();
```

Comment:
I am confused with cfs_opts and cf_options now. maybe we should think a better name for cf_options, I even prefer `v` here directly.